Uplift/Areas/Admin/Controllers/CategoryController.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Uplift.DataAccess.Data.Repository.IRepository;
+
+namespace Uplift.Areas.Admin.Controllers
+{
+    [Area("Admin")]
+    
+    public class CategoryController : Controller
+    {
+
+
+        private readonly IUnitOfWork _unitOfWork;
+
+        public CategoryController(IUnitOfWork unitOfWork)
+        {
+            _unitOfWork = unitOfWork;
```

Comment:
Did you fully get the dependency injection (DI) concept in C#?

Reply:
Yes .
i wanna record audio for you and send to you